### PR TITLE
elasticsearch 2.0

### PR DESCRIPTION
**Pull request by brainix**

### CODE REVIEW

File: Library/Formula/elasticsearch.rb [hunk 1]
```diff
@@ -47,12 +47,11 @@ def install
     # Set up Elasticsearch for local development:
     inreplace "#{prefix}/config/elasticsearch.yml" do |s|
       # 1. Give the cluster a unique name
-      s.gsub!(/#\s*cluster\.name\: elasticsearch/, "cluster.name: #{cluster_name}")
+      s.gsub!(/#\s*cluster\.name\: my-application/, "cluster.name: #{cluster_name}")
 
       # 2. Configure paths
       s.sub!(%r{#\s*path\.data: /path/to.+$}, "path.data: #{var}/elasticsearch/")
       s.sub!(%r{#\s*path\.logs: /path/to.+$}, "path.logs: #{var}/log/elasticsearch/")
-      s.sub!(%r{#\s*path\.plugins: /path/to.+$}, "path.plugins: #{var}/lib/elasticsearch/plugins")
```
Comment (bfontaine) [inline Library/Formula/elasticsearch.rb]: Where are plugins installed now?

Comment (brainix) [inline Library/Formula/elasticsearch.rb]: Good catch, @bfontaine, thank you.  The example config file no longer has a line to customize the plugin path, so I added the line to the end of the `brew` customized config file.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `elasticsearch 2.0.0`.

**Comment by mikemcquaid** [comment]
This doesn't seem to work correctly so I reverted it: https://github.com/Homebrew/homebrew/issues/45593

**Comment by mikemcquaid** [comment]
We should also probably improve the `test do` so this can't happen again.

**Comment by brainix** [comment]
Thanks for the heads up.  I'll work on writing more thorough tests and resubmit!

I believe that there are some permissions issues - my local environment may be different from others.  The formula works for me locally, but once you reopened this PR, I noticed that it's broken on my work computer.  Good catch, and apologies.

**Comment by jasontedor** [comment]
@brainix Thanks for lending a hand towards getting Elasticsearch 2.0.0 working with Homebrew. Unfortunately, there are some underlying issues that need to be worked out as discussed in #45644.

**Comment by brainix** [comment]
@jasontedor Thank you!  Yes, this formula quickly escalated beyond my depth of understanding of Homebrew.  I'll close this PR to avoid confusion, and the discussion can happen on #45644.

**Pull request closed without merging**